packages/app/src/components/applicationStages/style.css

```diff
@@ -0,0 +1,181 @@
+.nomination-bar-wrapper {
+    background: rgb(255, 255, 255);
+	margin: 1rem;
+	padding: 2rem;
```

Comment:
Nitpick: It looks like the indentation on these two lines and elsewhere in this file is off a bit. It looks like you may have been mixing spaces and tab characters.

Please stick with two spaces of indentation for consistency.